api/v1alpha1/virtualmachineclass_types.go

```diff
@@ -21,6 +21,34 @@ type DynamicDirectPathIODevice struct {
 	CustomLabel string `json:"customLabel,omitempty"`
 }
 
+// InstanceStorage provides information used to configure instance
+// storage volumes for a VM.
+//
+// StorageClass refers to the name of a StorageClass resource used to
+// provide the storage for the configured instance storage volumes.
+//
+// The value of this field has no relationship to or bearing on the field
+// virtualMachine.spec.storageClass.
+//
+// Please note the referred StorageClass must be available in the same
+// namespace as the VirtualMachineClass that uses it for configuring
+// instance storage.
+type InstanceStorage struct {
+	// StorageClass describes the configuration of instance storage related
+	// attributes on the VirtualMachine instance.
+	StorageClass string `json:"storageClass,omitempty"`
+
+	// Volumes describes the number and size of instance storage volumes
```

Comment:
Suggestion:

> // Volumes describes the instance storage volumes created for a VirtualMachine that uses the VirtualMachineClass.

A general note for elsewhere in the MR: we've typically avoided using abbreviations in the docs to be more precise about what is being referred to